S2VX.Game/Editor/S2VXEditor.cs

```diff
@@ -278,6 +294,22 @@ public class S2VXEditor : CompositeDrawable {
 
         private void PlaybackIncreaseBeatDivisor() => NotesTimeline.ChangeBeatDivisor(true);
 
+        private void PlaybackIncreaseRate() => PlaybackIncreaseRate(0.2);
+
+        private void PlaybackDecreaseRate() => PlaybackDecreaseRate(0.2);
+
+        public void PlaybackIncreaseRate(double step = 0.1) => Track.TempoTo(Math.Clamp(Track.Tempo.Value + step, 0.2, 1));
```

Comment:
It's a bit odd to me that Playback and Volume adjust at different rates depending if you use the menu or mouse wheel/key. I think these should change at the same amounts.